client/angular.json

```diff
@@ -71,6 +71,10 @@
         },
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
+          "options": {
+            "browserTarget": "client:build",
+            "proxyConfig": "src/proxy.conf.json"
```

Comment:
Why introduce the proxy?

Reply:
CORS failures that you have been experiencing caused by : localhost:4200 > localhost:8080

Reply:
I actually implemented a proxy to try to fix that but it didn't help so I removed it again.